### PR TITLE
TDP-5022 Improve Record vs StartRecording difference

### PR DESCRIPTION
**Pull request by rmello-daitan**
## For the Committer

⚠️ Ensure that for this repo (**bandwidth.github.io**) that the pull request change is opened against the branch `stop-gap-v2`

## Brief Summary of changes

People are getting confused about Record vs StartRecording use cases and sometimes don't even know of the existence of StartRecording verb. 

The intention is to mention the difference between the two at the top of each of those pages.

### CODE REVIEW

**Comment by marcelohossomi** [comment]
What about putting these tips in blue info boxes like we did at the end of [Transfer](https://dev.bandwidth.com/voice/bxml/verbs/transfer.html) page?
```
<aside class="alert general small"><p>There can be a maximum of 8 phone numbers to transfer to. </p></aside>
```

**Comment by rmello-daitan** [comment]
> What about putting these tips in blue info boxes like we did at the end of [Transfer](https://dev.bandwidth.com/voice/bxml/verbs/transfer.html) page?
> 
> ```
> <aside class="alert general small"><p>There can be a maximum of 8 phone numbers to transfer to. </p></aside>
> ```

Hmm it looks like I can't link the verb page inside that box. Is it still better?

**Comment by marcelohossomi** [comment]
> Hmm it looks like I can't link the verb page inside that box. Is it still better?

Really? Huh, so better leave like it's now.

File: voice/bxml/verbs/record.md [hunk 1]
```diff
@@ -4,6 +4,9 @@ The Record verb allows a segment of audio to be recorded during a call. At the e
 
 Bandwidth will keep recordings for up to 30 days. After 30 days the recordings will be deleted from Bandwidth's servers.
 
+<b>Tip:</b> This verb should probably be used if you are building a voice mail system, or conferencing introduction.
```
Comment (marcelohossomi) [inline voice/bxml/verbs/record.md]:
```suggestion
<b>Tip:</b> This verb should probably be used if you are building a voice mail system or conference member introduction.
```

File: voice/bxml/verbs/startRecording.md [hunk 2]
```diff
@@ -6,6 +6,8 @@ When used on a call, all audio on both sides of the call will be recorded.
 When used on a conference, all members who are not on mute or hold will be recorded, as well as all audio clips played.
 The recording will continue until the call or conference ends, the [`<StopRecording>`](stopRecording.md) verb is used, or the [`<PauseRecording>`](pauseRecording.md) verb is used.
 
+<b>Tip:</b> For use cases like voice mail system and conferencing introduction, you might want to take a look at [Record](../verbs/record.md).
```
Comment (marcelohossomi) [inline voice/bxml/verbs/startRecording.md]:
```suggestion
<b>Tip:</b> For use cases like voice mail system and conference member introduction, you might want to take a look at [Record](../verbs/record.md).
```

**Comment by muesono-daitan** [comment]
Maybe the preview site is not deploying because of the length of the branch name.
Can you use a shorter branch to see if it deploys the preview?

**Comment by rmello-daitan** [comment]
> Maybe the preview site is not deploying because of the length of the branch name.
> Can you use a shorter branch to see if it deploys the preview?

really?? let me see

**Pull request closed without merging**